### ISSUE TITLE
Distance checking doesn't work

### ISSUE THREAD

**Issue by ZenVee** [Qbox-project/qbx_police#87]
### Summary

isTooFar function in qbx_policejob/client/interactions.lua doesn't work

### Reproduction

1. Sign on duty has police office
2. Have player stand in from of you
3. type /jail or use the wheel menu to jail, search, escort

### Expected behavior

When performing one of the aforementioned actions there would be a distance check to make sure you're interacting with a player that's close instead of across the map. To do this you would take the coords returned from `lib.getClosestPlayer(coords)` and get the difference between them and your players coord to get the distance and then check that against a max distance to continue with the action.

### Actual behavior

`lib.getClosestPlayer(coords)` does not return distance, it is returning the coords of the player and we receive a "attempt to compare number with vector" error...

### Current Version

1.0.0

### Custom Resources

No

**Comment by solareon** [comment]
Can you give the f8 error that shows the compare number with vector?

There are only four instances of this in the current main branch and the code is looking to obtain the playerId or playerPed as appropriate.
![image](https://github.com/Qbox-project/qbx_policejob/assets/769465/9ddbd45c-b0de-4068-9ce7-1666959ca6d3)

Our function for the checks looks like this currently
https://github.com/Qbox-project/qbx_policejob/blob/c73064e25300cb59b37eee18743e3a682dc7ef51/client/interactions.lua#L141-L149

I put a patch together for the last instance of lib.getClosestPlayer that was incorrect [here](https://github.com/Qbox-project/qbx_policejob/tree/patch-distcheck)

**Comment by solareon** [comment]
@ZenVee I believe you might be on an older version. Try updating to the latest github commit and that problem should be gone. The `isTooFar` function is no longer part of the code

**Comment by ZenVee** [comment]
> @ZenVee I believe you might be on an older version. Try updating to the latest github commit and that problem should be gone. The `isTooFar` function is no longer part of the code

You're correct, and I'm an idiot lol. Sorry about that. I quickly checked the manifest for the version and seen it was the same so I assumed. I'll grab the update repo...

**Comment by solareon** [comment]
Not a problem at all. It did help us to catch another incorrect use of lib.getClosestPlayer

**Comment by ihyajb** [comment]
Fixed with an update but never closed, closing.
